Added fine tuned files

Galax-Git opened the pull request:
Includes leg mutations and a few organ mutations.

Commit:
Added fine tuned files

Commit:
Removed those pesky auto-tabs

AUTOMATIC1111 commented:
I think <affectsAllParts> should be true for lungs (although if you think not, leave it as is) and you should chance tabs to spaces.

Commit:
Changed lung mut.

Lung mutations now affect both lungs

Commit:
Tweaked files

Modified some mutations to compensate for having only one appedage mutated. Got rid of tabs.
